ReactCommon/cxxreact/Value.h

```diff
@@ -13,6 +13,7 @@
 #include <JavaScriptCore/JSValueRef.h>
 
 #include <folly/dynamic.h>
+#include <jni/LocalString.h>
```

Comment:
@ide, this already doesn't compile on ios since the header doesn't exist so I don't think that'd help